types/loud-rejection/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for loud-rejection 1.6
+// Project: https://github.com/sindresorhus/loud-rejection#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = loudRejection;
+
+declare function loudRejection(log?: (stack?: string) => void): void;
```

Comment:
`stack` should not be optional. the rational is, `loudRejection` just gets a callback, it will always call the callback with a `string`, the call back is free to ignore the argument though.

if that is not the case, then it should be `stack: string | undefined`